docs/user/openstack/install_upi.md

```diff
@@ -451,17 +466,18 @@ $ tree
 ### Remove Machines and MachineSets
 
 Remove the control-plane Machines and compute MachineSets, because we'll be providing those ourselves and don't want to involve the
-[machine-API operator][mao]:
+[machine-API operator][mao] and [cluster-control-plane-machine-set operator][ccpmso]:
```

Comment:
We don't have CCPMSO for openstack in 4.13, do we?

Reply:
I'll remove the commit, my bad.